rest/src/main/java/io/syndesis/rest/v1/handler/setup/OAuthAppHandler.java

```diff
@@ -128,7 +150,7 @@ private String getPropertyTaggedAs(Connector connector, String name) {
         return null;
     }
 
-    private void setPropertyTaggedAs(Connector connector, Map<String, String> configuredProperties, String name, String value) {
+    private static void setPropertyTaggedAs(Connector connector, Map<String, String> configuredProperties, String name, String value) {
         TreeSet EMPTY = new TreeSet();
```

Comment:
Sure it does :) [Collections.emptySortedSet()](https://docs.oracle.com/javase/8/docs/api/java/util/Collections.html#emptySortedSet--) (for new PR)